backend/app/models/model.py

```diff
@@ -0,0 +1,65 @@
+import os
```

Comment:
Lets break each class into its own file and rename each class/file to its class name (not under model)